entity/c/entity_client.c

```diff
@@ -0,0 +1,22 @@
+#include "c_api.h"
+
+int main() {
+    char path[] = "c_client.config";
```

Comment:
This should be passed through the main function's argument.

Reply:
Finished implementing in commit Finished implementing in commit https://github.com/iotauth/iotauth/commit/6f9fd165859b9fc198f30757c6e0341de59fe17a.